domains/pallets/executor-registry/src/tests.rs

```diff
@@ -77,7 +78,7 @@ impl pallet_balances::Config for Test {
 
 parameter_types! {
     pub const MinExecutorStake: Balance = 10;
-    pub const MaxExecutorStake: Balance = 1000;
+    pub const MaxExecutorStake: Balance = StakeWeight::MAX - 1;
```

Comment:
It is for convenience in `test_total_stake_overflow` where I want to simulate total stake overflow with a few executors and thus need to increase `MaxExecutorStake`. The `- 1` is because we also need to trigger the `StakeTooLarge` error in `register_should_work`.